tests/transforms/test_decompositions.py

```diff
@@ -754,7 +987,7 @@ def test_two_qubit_decomposition_3_cnots(self, U, wires):
         assert _compute_num_cnots(U) == 3
 
         obtained_decomposition = two_qubit_decomposition(U, wires=wires)
-        assert len(obtained_decomposition) == 10
+        assert len(obtained_decomposition) == 18
```

Comment:
Just to make sure why those changes are there. It is because previsouly we would get Rot instead of the decomposition?

Reply:
Yes. So now 3 gates are being returned instead of 1.